utils/helpers.js

```diff
@@ -130,6 +130,8 @@ export const getSample = (field) => {
   case 'email':
   case 'phone':
     return '';
+  case 'richText':
+    return JSON.stringify();
```

Comment:
getSample function is meant to return a sample field value. Here you are returning `undefined` which is probably not what you want.

Reply:
Oh yes thats true